app/assets/stylesheets/global.sass

```diff
@@ -123,50 +135,40 @@ html, body
           color: white
           background-color: $railsred
 
-      a
-        color: white
-        text-decoration: none
-
-    // Changes bootstraps default collapsing breakpoint
-    @media (min-width: $screen-xs) and (max-width: $screen-sm-max)
-      .navbar-collapse.collapse
-        display: none !important
-
-      .navbar-collapse.collapse.in
-        display: block !important
+    a
+      color: $white
+      text-decoration: none
 
-      .navbar-nav
+  @media screen and (max-width: $screen-md)
+    .navbar-collapse,
+    .navbar-inverse
+      background-color: transparent !important
+      background-image: none !important
+    .collapsing,
+    .collapse.in
+      border: none !important
+    li
+      background-color: transparent !important
+      a
+        background-color: transparent !important
+
+    .navbar-collapse.collapsing,
+    .navbar-collapse.collapse.in
+      border: 1px solid #ccc
+      border-right-color: #999
+      border-bottom-color: #999
+      margin-top: 10px
+      background-color: none
+      .nav
+        margin-left: auto
+        max-width: 200px
+      ul
         li
-          float: none
-
-  .divider
-    @media screen and (max-width: $screen-md)
-      border: 0.1px solid $railsred
-      width: 90%
-      margin-left: 5%
-    &:after
-      content: none !important
-
-
-  #menu-navbar
-    @media screen and (max-width: $screen-md)
-      z-index: 100
+          a
+            color: $railsred
 
   .navbar-toggle
```

Comment:
For some reason, there's now a side effect: the burger icon is getting a black border. Not exactly sure where it comes from 🤔 but a `border: none` should fix it..